tt/actions/read/report.py

```diff
@@ -7,37 +7,44 @@
 from tt.actions.utils import reportingutils
 
 
-def action_report(colorizer, activity):
-    print('Displaying all entries for ', colorizer.yellow(activity), ' grouped by day:', sep='')
+def action_report(colorizer, activity=None):
+    if activity is None:
+        print('Displaying all entries grouped by day:', sep='')
+    else:
+        print('Displaying all entries for ', colorizer.yellow(activity), ' grouped by day:', sep='')
     print()
     sep = ' | '
     data = get_data_store().load()
     work = data['work']
-    report = defaultdict(lambda: {'sum': timedelta(), 'notes': '', 'weekday': '', 'start_time': None, 'end_time': None})
+    report = defaultdict(lambda: {'sum': timedelta(), 'notes': [], 'weekday': '', 'start_time': None, 'end_time': None})
 
     total_time = 0
     for item in work:
-        if item['name'] == activity and 'end' in item:
+        if (item['name'] == activity or activity is None) and 'end' in item:
             start_time = parse_isotime(item['start'])
             end_time = parse_isotime(item['end'])
             day = reportingutils.extract_day(item['start'])
             duration = parse_isotime(item['end']) - parse_isotime(item['start'])
             report[day]['sum'] += duration
-            report[day]['notes'] += reportingutils.get_notes_from_workitem(item)
+            if activity is None:
+                notes = reportingutils.get_notes_from_workitem(item)
+                report[day]['notes'].append(item['name'] + (f": {notes}" if notes else ""))
```

Comment:
How about printing the work item name in square brackets like [work_item]

`report[day]['notes'].append("[" + item['name'] + (f"] {notes}" if notes else ""))`